ibazel/ibazel.go

```diff
@@ -128,13 +136,22 @@ func (i *IBazel) SetBazelArgs(args []string) {
 	i.bazelArgs = args
 }
 
+func (i *IBazel) SetNoLiveReload(noLiveReload bool) {
+	i.noLiveReload = noLiveReload
+}
+
+func (i *IBazel) SetProfileDev(profileDev bool) {
+	i.profileDev = profileDev
+}
+
 func (i *IBazel) Cleanup() {
 	i.buildFileWatcher.Close()
```

Comment:
It seems like there should be a step where the server gets shut down in here. If not, can you help me understand why?